docs/source/workflows/editor-plugins.md

```diff
@@ -1,65 +1,61 @@
 ---
-title: Connecting Apollo to your editor
+title: Editor plugins
```

Comment:
Can we make this more actionable? `GraphQL-aware Editor`, `GraphQL insights in Editor`, or `Enabling GraphQL in Editor`